server/config/config.go

```diff
@@ -241,7 +241,7 @@ var (
 	defaultRuntimeServices = []string{}
 	defaultLocationLabels  = []string{}
 	// DefaultStoreLimit is the default store limit of add peer and remove peer.
-	DefaultStoreLimit StoreLimit = StoreLimit{AddPeer: 15, RemovePeer: 15}
```

Comment:
Why change it to 2000?

Reply:
Just for current bench test, he will change it back.